docs/assets/trino/pom.xml

```diff
@@ -22,6 +21,27 @@
     </properties>
 
     <dependencies>
+
+        <dependency>
+            <groupId>com.datastrato.gravitino</groupId>
+            <artifactId>bundled-catalog</artifactId>
+            <version>0.5.0-SNAPSHOT</version>
+        </dependency>
+
+        <!--
+        The following dependencies are required for the Gravitino connector. You can install them
+        locally (./gradlew publishToMavenLocal) or just use the release version like 0.5.0
```

Comment:
Can you please modify `0.5.0` to `0.4.0` as version `0.5.0` has not released? It's a mistake that I made.